sdk/quantum/quantum/rollup.config.js

```diff
@@ -12,7 +12,7 @@ import sourcemaps from "rollup-plugin-sourcemaps";
 
 /** @type {rollup.RollupFileOptions} */
 const config = {
-  input: "./dist-esm/quantumClient.js",
+  input: "./dist-esm/quantumJobClient.js",
   external: ["@azure/core-http"],
   output: {
     file: "./dist/quantum.js",
```

Comment:
Shouldnt the output file be updated as well? Same for the `name` field

Reply:
Modified output and name fields